src/mainboard/clevo/adl-p/vboot-rwa.fmd

```diff
@@ -0,0 +1,40 @@
+FLASH@0xff000000 0x1000000 {
+	SI_ALL 5M {
+		SI_DESC 4K
+		SI_ME
+	}
+
+	RW_MISC 400K {
+		UNIFIED_MRC_CACHE(PRESERVE) 128K {
+			RECOVERY_MRC_CACHE 64K
+			RW_MRC_CACHE 64K
+		}
+		SMMSTORE(PRESERVE) 256K
+		RW_SHARED 0x4000 {
+			SHARED_DATA@0x0 0x2000
+			VBLOCK_DEV@0x2000 0x2000
+		}
+	}
+
+	CONSOLE 0x20000
+	BOOTSPLASH(CBFS) 1M
+
+	RW_SECTION_A {
+		VBLOCK_A 0x2000
+		FW_MAIN_A(CBFS)
+		RW_FWID_A 0x40
+	}
+
+	# T/B = 0
+	# BP3 = 0 BP2 = 1 BP1 = 1 BP0 = 1
```

Comment:
These two comments can be removed. We're using flash protection features from the chipset, not from the actual SPI chip.

Reply:
Removed.